source/index.html.md

```diff
@@ -673,6 +673,38 @@ object][ApplePay-PaymentToken] for more information.
   strongly authenticated (SCA in PSD2).
 </p>
 
+##### Method: `token`
+
+<dl class="dl-vertical">
+  <dt>token[tpan]
+    <span class="type">[0-9]{12,19}</span>
+  </dt>
+  <dd>
+    Primary account number of card to charge.
+  </dd>
+  <dt>token[expire_month]
+    <span class="type">[0-9]{2}</span>
+  </dt>
+  <dd>Expiry month of card to charge.</dd>
+  <dt>token[expire_year]
+    <span class="type">20[0-9]{2}</span>
+  </dt>
+  <dd>Expiry year of card to charge.</dd>
+  <dt>token[tav]
+    <span class="type">[:base64:]{19,21}</span>
+  </dt>
+  <dd>Token Authentication Value</dd>
+  <dt>token[payment_method_indication]
+    <span class="type">[\x20-\x7E]{7,15}
+      <a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Indicate payment method used. Either: <code>applepay</code>, <code>samsungpay</code>, <code>mobilepayonline</code>, <code>unknown</code>
```

Comment:
An alternative to `unknown` could be to make this optional.

Perhaps we should hint what it is used for; why it could be beneficial to tell us about the payment method.